alf/utils/data_buffer.py

```diff
@@ -506,6 +506,9 @@ def get_batch_by_indices(self, indices):
                                             self._derived_buffer)
         return convert_device(result)
 
+    def is_full(self):
+        return (self.current_size == self._capacity).cpu().numpy()
+
```

Comment:
Please note that this will return an array of 1 element?

Reply:
This function is not used?

Reply:
> Please note that this will return an array of 1 element?

It's not an array. self.current_size is a function that uses self._current_size[0]

Reply:
> This function is not used?

not used here. But to summarize embeddings, potentially we need a buffer to store many of them and summarize when it's full.

Reply:
> > Please note that this will return an array of 1 element?
> 
> It's not an array. self.current_size is a function that uses self._current_size[0]

Try tensor([0]).numpy(), it returns an array(0).

Reply:
> > > Please note that this will return an array of 1 element?
> > 
> > 
> > It's not an array. self.current_size is a function that uses self._current_size[0]
> 
> Try tensor([0]).numpy(), it returns an array(0).

I mean self.current_size is torch.tensor(0) (self._current_size[0]) instead of torch.tensor([0]).

Reply:
I see, scalar tensors can be automatically converted into float and bool.  LG.